chapter_01/p01_is_unique.py

```diff
@@ -24,6 +24,21 @@ def is_unique_chars_pythonic(string):
     return len(set(string)) == len(string)
 
 
+def is_unique_no_ds(string):
```

Comment:
My read of the book answer is that it considers the bit-vector `checker` variable to be a data structure so this would be better named `is_unique_bit_vector` or something similar.

Reply:
Yes, you're right. I changed the name to `is_unique_bit_vector `: https://github.com/careercup/CtCI-6th-Edition-Python/pull/150/commits/cd95b439aa76d7f3e765ac7473f1ccc28b7949fa